ext_emconf.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * (c) 2020 FriendsOfCrawler <mail@treupo.de>
+ *
+ * This file is part of the TYPO3 Crawler Widgets Extension.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+$EM_CONF[$_EXTKEY] = [
+    'title' => 'TYPO3 Crawler Widgets',
+    'description' => 'Widgets for crawler extension',
+    'category' => 'be',
+    'constraints' => [
+        'depends' => [
+            'typo3' => '^10.4',
```

Comment:
'typo3' => '10.4.0-10.4.99' - etc.

Reply:
@brotkrueml @tomasnorre 
I changed the versions for the typo3 packages but I am not sure about the aoepeople/crawler package. Is 9.0.0-9.9.99 enough?